test/fixtures/service/socket/main.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
```

Comment:
This file will be installed as a binary in one of the test systemd spec containers, but the server it starts won't actually be checked. I'm not sure we can really do a full e2e test in our current setup since systemd must already be running in a container for the server to be initialized